mix.exs

```diff
@@ -123,6 +128,54 @@ defmodule Sentry.Mixfile do
   end
 
   defp aliases do
-    [test: ["sentry.package_source_code", "test"]]
+    [
+      test: ["sentry.package_source_code", "test", &run_integration_tests_if_supported/1],
+      "test.integrations": &test_integrations/1
+    ]
+  end
+
+  defp run_integration_tests_if_supported(_args) do
+    if Version.match?(System.version(), ">= 1.16.0") do
+      Mix.Task.run("test.integrations")
+    else
+      IO.puts("Skipping integration tests for Elixir versions < 1.16")
+    end
+  end
+
+  @integrations [
+    "phoenix_app"
+  ]
+
+  defp test_integrations(args) do
+    if Version.match?(System.version(), ">= 1.16.0") do
+      for integration <- @integrations do
+        run_integration_tests(integration, args)
+      end
+    else
+      IO.puts("Skipping integration tests for Elixir versions < 1.16")
+    end
+  end
+
+  defp run_integration_tests(integration, args) do
+    header = if IO.ANSI.enabled?(), do: IO.ANSI.bright() <> IO.ANSI.cyan(), else: ""
+    reset = if IO.ANSI.enabled?(), do: IO.ANSI.reset(), else: ""
```

Comment:
You can use `IO.ANSI.format/1`, whose second (default) argument is `IO.ANSI.enabled?()`, so that it magiworks and you can do

```elixir
IO.puts(IO.ANSI.format(["\n", :bright, :cyan, "==> Running tests for integration: #{integration}"])
```

(also resets at the end)

Reply:
Same below btw